packages/yarnpkg-cli/sources/main.ts

```diff
@@ -60,12 +60,19 @@ export async function main({binaryVersion, pluginConfiguration}: {binaryVersion:
     // Non-exhaustive known requirements:
     // - 10.16+ for Brotli support on `plugin-compat`
     // - 10.17+ to silence `got` warning on `dns.promises`
+    // - 14.0 and 14.1 empty http responses - https://github.com/sindresorhus/got/issues/1496
 
     const version = process.versions.node;
     const range = `>=10.17`;
```

Comment:
Why not just update this line?

Reply:
To have a more specific error message

Reply:
It doesn't matter too much for users why a platform isn't supported, so I'd prefer to forego the message and use a single range